test/parallel/test-cli-options.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+if (module.parent) {
+  // signal we've been loaded as a module
+  console.log('Loaded as a module, exiting with status code 42.');
+  process.exit(42);
+}
+
+const common = require('../common');
+const assert = require('assert');
+const child = require('child_process');
+const path = require('path');
+const nodejs = '"' + process.execPath + '"';
+
+child.exec(nodejs +
+    ' --no-dom-globals --eval "process.stdout.write(typeof setTimeout)"',
+    function(err, stdout, stderr) {
```

Comment:
Could you `assert.ifError(err)` for completeness. And maybe change `equal()` to `strictEqual()`.